lib/code_corps/model/role.ex

```diff
@@ -23,8 +23,13 @@ defmodule CodeCorps.Role do
   @doc """
   Builds a changeset based on the `struct` and `params`.
   """
-  @spec changeset(CodeCorps.Role.t, map) :: Ecto.Changeset.t
-  def changeset(struct, params \\ %{}) do
+  @spec create_changeset(Codecorps.Role.t, map) :: Ecto.Changeset.t
+  def create_changeset(struct, params \\ %{}) do
+    changeset(struct, params)
+  end
```

Comment:
I see you did the same across several PRs. Sorry for not reviewing earlier and saving you some time, but I think it would be preferable to revert on this and simply call `Role.changeset` from the controller instead.